spec/controllers/passwords_controller_spec.rb

```diff
@@ -4,9 +4,10 @@
   describe "POST email" do
     context "with valid email address" do
       it "should send an email with the correct token" do
-        user = Fabricate(:user)
-        post :email, params: { email: user.email }
+        Fabricate(:user, email: "example@example.com")
```

Comment:
The original, automatic email is a common pattern. Either way works, but you will not usually specify user data for a fabrication unless there is a special circumstance.